[MRG] Update contributing.md (#897)
jupyterhub/team-compass: docs/contribute/guide.md

* [![Good First Issue](https://img.shields.io/badge/-good%20first%20issue-blueviolet.svg)][link_goodfirstissue] *These issues contain a task that a member of the team thinks could be a good entry point to the project.*

## Guidelines to getting a Pull Request merged

These are not strict rules, but recommended guidelines from the JupyterHub maintainers to help make your contribution process as smooth and effective as possible for both you and the community.
- **Create a PR as early as possible**, marking it with `[WIP]` while

>  This helps prevent duplicated work, allows maintainers to give early feedback on design or API changes, and can attract collaborators.to work with you.

Reviewer: ```suggestion
 This helps prevent duplicated work, allows maintainers to give early feedback on design or API changes, and can attract collaborators to work with you.
```

- **Keep your PR focused.**
  The best PRs address a single issue or feature.
  If you end up changing multiple things, please open separate PRs for the different conceptual changes.
- **Add tests to your code.**
  Contributions should come with appropriate tests and all continuous intergration (CI) checks must pass before the PR can be merged.

>   PRs will not be merged if Travis is failing.

Reviewer: ```suggestion
  PRs will not be merged if tests are failing.
```

> - **Apply [PEP8](https://www.python.org/dev/peps/pep-0008/)** as much as possible, but not too much.

Reviewer: ```suggestion
- **Apply [PEP8](https://www.python.org/dev/peps/pep-0008/)** principles, but no need to be overly strict. Most of our repos have automatic formatting via [pre-commit](https://pre-commit.com/)
```
